### PR TITLE
Fix/condo/sberdoma 701/use employee user values

### PR DESCRIPTION
**Pull request by nomerdvadcatpyat**
reopen pr #362

When user updates his name, email or phone, employees in his organization can view updated data

### CODE REVIEW

File: apps/condo/domains/user/schema/User.js [hunk 1]
```diff
@@ -130,6 +131,30 @@ const User = new GQLListSchema('User', {
...
     },
+    hooks: {
+        afterChange: async ({ updatedItem, context, existingItem, operation }) => {
+            if (
+                operation === 'update' && existingItem &&
+                (updatedItem.phone !== existingItem.phone ||
+                updatedItem.email !== existingItem.email ||
+                updatedItem.name !== existingItem.name)
+            ) {
+                const employees = await OrganizationEmployee.getAll(context, { user: { id: updatedItem.id } })
```
Comment (pahaz) [inline apps/condo/domains/user/schema/User.js:142]: it's better to filter by isAccepted here

File: apps/condo/domains/user/schema/User.js [hunk 2]
```diff
@@ -130,6 +131,30 @@ const User = new GQLListSchema('User', {
         },
 
     },
+    hooks: {
+        afterChange: async ({ updatedItem, context, existingItem, operation }) => {
+            if (
```
Comment (pahaz) [inline apps/condo/domains/user/schema/User.js:136]: it's better to group this code by creating a function `updateEmployeesRelatedToUser`

**Review by pahaz — APPROVED** [review]

File: apps/condo/domains/organization/schema/OrganizationEmployee.test.js [hunk 3]
```diff
@@ -231,4 +237,91 @@ describe('OrganizationEmployee', () => {
             await updateTestOrganizationEmployee(userClient, obj.id, { name: 'name3' })
         })
     })
+
```
Comment (pahaz) [inline apps/condo/domains/organization/schema/OrganizationEmployee.test.js:240]: You need to add a test for after change hook

Comment (nomerdvadcatpyat) [reply apps/condo/domains/organization/schema/OrganizationEmployee.test.js:240]: it is tested in these tests, for example in test 'employee who accepted the invite: updates phone and email if employee user updates' I tested that when updating the user we updates relates employee (in after change hook)

**Review by dkoviazin — APPROVED** [review]

**Pull request merged**